### PR TITLE
Esplugues / Import geo datasets

### PR DESCRIPTION
**Pull request by jbb231**
Closes [#1775](https://github.com/PopulateTools/issues/issues/1775)

- Create pipelines for geo-paradas and geo-equipamientos. 
- Create datasets schemas and transform_template.sql scripts

### CODE REVIEW

File: pipelines/geo-equipamientos/dev_run.sh [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+
+# Clean working dir
+cd $ETL_UTILS; ruby operations/prepare-working-directory/run.rb $WORKING_DIR
+
+# Download data in CSV format
+table_list="atencio_social cultura ensenyament esplais_jubilats esport mercats residencies_geriatriques salut"
+
+for table in $table_list; do 
+    echo "Processing $table"
+    cd $ETL_UTILS; ruby operations/download/run.rb "https://mun.nexusgeographics.com/geoserver/esplugues/ows?service=WFS&version=1.0.0&request=GetFeature&typeName=esplugues%3A${table}&maxFeatures=10000&outputFormat=csv" $WORKING_DIR/${table}_raw.csv
```
Comment (ferblape) [inline pipelines/geo-equipamientos/dev_run.sh:24]: @jbb231 this line is donwloading the data to WORKING_DIR

You have defined it as `/tmp/geo_equipamientos` in line 10

Comment (jbb231) [reply pipelines/geo-equipamientos/dev_run.sh:24]: yes, but I cannot see this folder in my computer, is that normal?

File: pipelines/geo-equipamientos/dev_run.sh [hunk 2]
```diff
@@ -0,0 +1,44 @@
+#!/bin/bash
+
+# Documentation 
+#
+# https://drive.google.com/file/d/1U23HS_EQQwv8C-mcC-0rjAQiFFA_kJI2/view
+
+set -e
+source .env
```
Comment (ferblape) [inline pipelines/geo-equipamientos/dev_run.sh:8]: This env file is not required as  far as I can see, the variables you are defining on it are not used.

File: pipelines/geo-paradas/dev_run.sh [hunk 3]
```diff
@@ -0,0 +1,44 @@
+#!/bin/bash
+
+# Documentation 
+#
+# https://drive.google.com/file/d/1U23HS_EQQwv8C-mcC-0rjAQiFFA_kJI2/view
+
+set -e
+source .env
```
Comment (ferblape) [inline pipelines/geo-paradas/dev_run.sh:8]: The same

File: pipelines/geo-equipamientos/dev_run.sh [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+WORKING_DIR=/tmp/geo_equipamientos
+ETL_UTILS=$DEV_DIR/gobierto-etl-utils
+ETL=$DEV_DIR/gobierto-etl-esplugues
+GOBIERTO_URL=$1
+
+# Clean working dir
+cd $ETL_UTILS; ruby operations/prepare-working-directory/run.rb $WORKING_DIR
+
+# Download data in CSV format
+table_list="bus trambaix metro"
```
Comment (ferblape) [inline pipelines/geo-equipamientos/dev_run.sh:19]: Careful, the table names should be different

File: pipelines/geo-paradas/Jenkinsfile [hunk 5]
```diff
@@ -0,0 +1,73 @@
...
+        }
+        stage('Load') {
+          steps {
+            sh '''#!/bin/bash
+              source ${ETL}/.rbenv-vars;
+              cd ${ETL_UTILS};
+              bundle exec ruby operations/gobierto_data/upload-dataset/run.rb \
+                    --api-token $GOBIERTO_API_TOKEN \
+                    --gobierto-url ${GOBIERTO_URL} \
+                    --name "Paradas" \
```
Comment (ferblape) [inline pipelines/geo-paradas/Jenkinsfile:54]: Let's call it "Paradas de Transporte Público"

File: datasets/geo-equipamientos/schema.json [hunk 6]
```diff
@@ -0,0 +1,10 @@
+{
+    "id":{"type":"text"},
+    "equipment_name":{"type":"text"},
+    "equipment_type":{"type":"text"},
+    "location":{"type":"text"},
+    "web":{"type":"text"},
+    "phone_number":{"type":"text"},
+    "geom":{"type":"text"}
```
Comment (ferblape) [inline datasets/geo-equipamientos/schema.json:8]: Could. you rename this column to `geometry`?

File: datasets/geo-paradas/schema.json [hunk 7]
```diff
@@ -0,0 +1,8 @@
+{
+    "id":{"type":"text"},
+    "name_stop":{"type":"text"},
+    "transport_type":{"type":"text"},
+    "location":{"type":"text"},
+    "geom":{"type":"text"}
```
Comment (ferblape) [inline datasets/geo-paradas/schema.json:6]: Could. you rename this column to `geometry`?

File: pipelines/geo-equipamientos/Jenkinsfile [hunk 8]
```diff
@@ -0,0 +1,74 @@
...
+            script {
+              for (table in ${table_list}) {
+                url = "https://mun.nexusgeographics.com/geoserver/esplugues/ows?service=WFS&version=1.0.0&request=GetFeature&typeName=esplugues%3A"${table}"&maxFeatures=10000&outputFormat=csv"
+                sh '''#!/bin/bash
+                   cd ${ETL_UTILS};
+                   bundle exec ruby operations/download/run.rb  ${url} ${WORKING_DIR}/${table}_raw.csv
+                   ruby operations/load-csv-sqlite/run.rb ${WORKING_DIR}/${table}_raw.csv ${WORKING_DIR}/data.sqlite "," ${table}_raw 
+                   '''
+                    }
+            }
```
Comment (ferblape) [inline pipelines/geo-equipamientos/Jenkinsfile:30]: Could you try these syntax?

```suggestion
            sh '''#!/bin/bash
              for table in ${table_list}; do
                echo "Processing $table"
                cd $ETL_UTILS; ruby operations/download/run.rb "https://mun.nexusgeographics.com/geoserver/esplugues/ows?service=WFS&version=1.0.0&request=GetFeature&typeName=esplugues%3A${table}&maxFeatures=10000&outputFormat=csv" $WORKING_DIR/${table}_raw.csv
                ruby operations/load-csv-sqlite/run.rb $WORKING_DIR/${table}_raw.csv $WORKING_DIR/data.sqlite ',' ${table}_raw
              done
            '''
```

**Pull request merged**